cpp/velox/operators/functions/RegistrationAllFunctions.cc

```diff
@@ -67,6 +68,10 @@ void registerFunctionOverwrite() {
       kRowConstructorWithAllNull,
       std::make_unique<RowConstructorWithNullCallToSpecialForm>(kRowConstructorWithAllNull));
   velox::functions::sparksql::registerBitwiseFunctions("spark_");
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedPlusFunction>({"try_add"});
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedMinusFunction>({"try_subtract"});
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedMultiplyFunction>({"try_multiply"});
+  velox::functions::registerBinaryIntegral<velox::functions::CheckedDivideFunction>({"try_divide"});
```

Comment:
I notice they are registered in Velox Presto functions. Can we also register them in Velox Spark functions?

Reply:
Yes, if we can support them in Velox Spark functions, we could remove the custom registration in Gluten.

Reply:
We can firstly merge this pr. Then remove the registering after velox pr is merged.